docs/scripts/schemas/unversioned/eas-json-build-common-schema.js

```diff
@@ -40,6 +40,14 @@ export default [
       '[Learn more about custom development clients](../../clients/introduction).',
     ],
   },
+  {
+    name: 'prebuildCommand',
+    type: 'string',
+    description: [
+      'Optional override of the prebuild command used by EAS.',
```

Comment:
consider adding example value to clarify how to prebuild command should be passed, we support multiple ways to define it but I would suggest sth like this `prebuild --template example-template`